appium/sample-scripts/python/device_finder.py

```diff
@@ -100,11 +100,11 @@ def device_API_level(self, deviceName):
         print "Searching for API level of device '%s'" % deviceName
 
         try:
-            device = self.get(path="devices?search=%s" % deviceName)
-            apiLevel = device['data'][0]['softwareVersion']['apiLevel']
-            print "Selected device '{}'".format(device['data'][0]['displayName'])
-            print "Found API level: %s" % apiLevel
-            return apiLevel
+            devices = self.get(path="devices?search=%s" % deviceName)['data']
+            shuffle(devices)
+            picked_device = devices[0]
+            print "Selected device '{}' has API level '{}'".format(picked_device['displayName'], picked_device['softwareVersion']['apiLevel'])
+            return picked_device
```

Comment:
method named "device_API_level" returns device? 
"Find out the API level of a Device"